src/Voron/Data/Tables/Table.cs

```diff
@@ -727,21 +729,41 @@ private void ThrowBadAllocation(int size)
 
         public class CompressionDictionariesHolder : IDisposable
         {
-            private readonly ConcurrentDictionary<int, ZstdLib.CompressionDictionary> _compressionDictionaries = new ConcurrentDictionary<int, ZstdLib.CompressionDictionary>();
+            public ConcurrentDictionary<int, ZstdLib.CompressionDictionary> CompressionDictionaries { get; } = new();
```

Comment:
why? readonly field vs property in this case might have a significant impact on performance

Reply:
I reverted it and implemented the functionality through methods.